.tx/config

```diff
@@ -9,7 +9,7 @@ file_filter = MapboxNavigation/Resources/<lang>.lproj/Localizable.strings
 source_file = MapboxNavigation/Resources/Base.lproj/Localizable.strings
 source_lang = en
 
-[mapbox-navigation-core-ios.localizablestrings]
+[mapbox-navigation-ios.LocalizableCoreStrings]
 file_filter = MapboxCoreNavigation/Resources/<lang>.lproj/Localizable.strings
```

Comment:
I removed `core` from `mapbox-navigation-core-ios` to put them in the same project but had to rename the slug `localizablestrings` since it's already being used in `mapbox-navigation-ios`. I'm not sure if I can rename the other slugs to CamelCase for consistency (if we loose the translations) but there are no duplicates here. (We use "localizablestrings" for the UI & "LocalizableCoreStrings" for Core)